### DISCUSSION TITLE
Project (beta): is 'Export' a feature on the roadmap?

### DISCUSSION THREAD

**Discussion by CodeCox** [community/community#5411] [Projects and Issues] [answered]
The Project (beta) Table view is spreadsheet-like so exporting the data to a spreadsheet sounds like a natural extension to abet data sharing. It could support formats `.xlsx` for spreadsheet , `.csv` for text, and `html' for docs?

I spotted a mention of GraphQL & API support somewhere - I guess that will be another route to getting this data for external use?

**Comment by queenofcorgis** [comment, marked as answer]
You can now export your project view to a CSV file! Check it out on the [changelog](https://github.blog/changelog/2023-06-15-github-issues-projects-june-15th-update/#%F0%9F%8F%97%EF%B8%8F-export-project-view-as-a-csv-file) announcement and let us know what you think 🚀

**Comment by royarzun95** [comment]
is there any way to export the roadmap with dates?
